### PR TITLE
Misc test fixups

### PR DESCRIPTION
**Pull request by joestringer**
Fix up some spelling, copyright headers in the test dir.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7496)

### CODE REVIEW

**Review by aanm — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

File: test/ginkgo-ext/scopes.go [hunk 1]
```diff
@@ -257,7 +258,7 @@ func runAllAfterFail(cs *scope, testName string) {
 		for _, body := range cs.afterFail {
 			body()
 		}
-		GinkgoPrint("===================== Existing AfterFailed =====================")
```
Comment (nebril) [inline test/ginkgo-ext/scopes.go:260]: I liked this one ;)

**Review by nebril — APPROVED** [review]

**Comment by borkmann** [comment]
test-me-please

**Review by eloycoto — APPROVED** [review]

**Comment by ianvernon** [comment]
test-me-please

File: test/ginkgo-ext/scopes.go (hunk 1 above)
Comment (joestringer) [reply test/ginkgo-ext/scopes.go:260]: I almost wanted to keep it for posterity:)

**Comment by joestringer** [comment]
Hit #7373:

https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/11071/testReport/junit/k8s-1/10/K8sDatapathConfig_Encapsulation_Check_connectivity_with_Geneve_encapsulation/

**Comment by joestringer** [comment]
test-me-please

EDIT: VM download failure...
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/11078/execution/node/57/log/

**Comment by joestringer** [comment]
test-me-please

**Pull request merged**